db/db.go

```diff
@@ -42,7 +42,7 @@ func (db *DB) GetResource(id manifold.ID) *Resource {
 func (db *DB) DeleteResource(id manifold.ID) bool {
 	_, ok := db.ResourcesByID[id]
 	if ok {
-		cs, _ := db.CredentialsByResource[id]
+		cs := db.CredentialsByResource[id]
```

Comment:
So this will now panic if the ID isn't present. Before, the `cs` would have just been an empty slice (and thus skip the loop).

If linting fails because we ignore the boolean check, I'd suggest doing a check based on that to do the loop or not, because we don't want to panic in this case.

Reply:
Apparently maps don't panic when you don't use the check! nvm then

Reply:
As explained in chat, maps don't panic like typecasting